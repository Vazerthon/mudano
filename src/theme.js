const theme = {
  colour: {
    primary: '#1e252b',
    secondary: '#dcd6ca',
    highlight: '#9a3cbb',
    timeline: {
      P: '#0971b2', // public holiday
      V: '#54cc14', // vacation
      T: '#cc4a14', // training
      present: '#9a3cbb',
    },
  },
  layout: {
    padding: '8px',
  },
};

export default theme;
