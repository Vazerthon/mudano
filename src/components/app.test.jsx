/* eslint-env jest */

import React from 'react';
import shallowThemeWrapper from './test-helper';
import theme from '../theme';

import App from './App';

const shallow = component => shallowThemeWrapper(component, theme);

describe('app component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <App stageNewEntry={() => {}} submitEntry={() => {}} />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
