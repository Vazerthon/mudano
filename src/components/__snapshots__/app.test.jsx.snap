// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app component renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App
    stageNewEntry={[Function]}
    stagedEntry={Object {}}
    submitEntry={[Function]}
    timeline={Array []}
    user={Object {}}
    users={Array []}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <styled.h1>
          Mudano / 
          <styled.span>
                    Absentee Manager
          </styled.span>
</styled.h1>,
        <Timeline
          timeline={Array []}
          user={Object {}}
          users={Array []}
/>,
        <EntryMaker
          entry={Object {}}
          stageNewEntry={[Function]}
          submitEntry={[Function]}
/>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            "Mudano / ",
            <styled.span>
              Absentee Manager
</styled.span>,
          ],
        },
        "ref": null,
        "rendered": Array [
          "Mudano / ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "Absentee Manager",
            },
            "ref": null,
            "rendered": "Absentee Manager",
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "timeline": Array [],
          "user": Object {},
          "users": Array [],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "entry": Object {},
          "stageNewEntry": [Function],
          "submitEntry": [Function],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <styled.h1>
            Mudano / 
            <styled.span>
                        Absentee Manager
            </styled.span>
</styled.h1>,
          <Timeline
            timeline={Array []}
            user={Object {}}
            users={Array []}
/>,
          <EntryMaker
            entry={Object {}}
            stageNewEntry={[Function]}
            submitEntry={[Function]}
/>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              "Mudano / ",
              <styled.span>
                Absentee Manager
</styled.span>,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Mudano / ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Absentee Manager",
              },
              "ref": null,
              "rendered": "Absentee Manager",
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "timeline": Array [],
            "user": Object {},
            "users": Array [],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "entry": Object {},
            "stageNewEntry": [Function],
            "submitEntry": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "context": Object {
      "__styled-components__": [Function],
      "__styled-components__next__": Object {
        "getTheme": [Function],
        "subscribe": [Function],
        "unsubscribe": [Function],
      },
    },
  },
}
`;
